dali/operators/nvcvop/nvcvop.cc

```diff
@@ -147,4 +160,34 @@ void PushImagesToBatch(nvcv::ImageBatchVarShape &batch, const TensorList<GPUBack
   }
 }
 
+nvcv::Tensor AsTensor(const Tensor<GPUBackend> &tensor, TensorLayout layout = "",
+                      const std::optional<TensorShape<>> &reshape = {}) {
```

Comment:
I moved the default values to the definition and used nullopt instead of {}.

Regarding the optional vs nullable pointer: I prefer optional, it's safer imo